scripts/liveMonitor_countTests/count-java-tests.py

```diff
@@ -0,0 +1,84 @@
+from sys import argv
+import os
+import glob
+
+if len(argv) != 2:
+    print("Usage: count-java-tests.py [directory]")
+    exit()
+
+searchDir = argv[1]
+if not os.path.isdir(searchDir):
+    print("Error! '" + searchDir + "' is not a directory")
+    exit()
+
+print("\nCounting tests in '" + searchDir + "' ...")
+
+def get_filepaths(directory, extension):
+    file_paths = []
+    for root, directories, files in os.walk(directory):
+        for filename in files:
+            if filename.endswith(extension):
+                filepath = os.path.join(root, filename)
+                file_paths.append((root, filepath, filename))
+    return file_paths
+
+# Run the above function and store its results in a variable.   
+full_file_paths = get_filepaths(searchDir, "java")
+
+print("\nFound " + str(len(full_file_paths)) + " java files\n")
+
+searchDirParts = searchDir.split('/')
+
+# count per directory and running count
+count = 0
+paths = {}
+for root, filepath, filename in full_file_paths:
+    fopen = open(filepath,mode='r')
+    found = False
+    fread = fopen.readlines()
+    for line in fread:
+        if '@test' in line:
+            found = True
+            break
+
+    if found:
+        count = count + 1
+        subDirParts = root.split('/')
+        if len(subDirParts) == len(searchDirParts):
+            subdir = subDirParts[len(subDirParts) - 1]
+        elif len(subDirParts) > len(searchDirParts):
+            subdir = "./" + subDirParts[len(searchDirParts)]
```

Comment:
regarding https://github.com/AdoptOpenJDK/openjdk-tests/pull/2498/files#r613474627, there is actually something you need to modify. Replace line 46 - 50 to `subdir = os.path.relpath(root, searchDir)` also remove line 30. This should work.

Reply:
Yes, this worked.